examples/common/common.yaml

```diff
@@ -49,10 +49,11 @@ spec:
         spec:
           properties:
             virtualhost:
+              required:      
+                - fqdn
```

Comment:
I think making this required breaks any non-root ingressroutes.

Reply:
Thanks for spotting that. Maybe we just drop the pattern all together. At best all I think we can say about this field is, if it is present, it shouldn't be blank. Which is basically saying nothing.

Reply:
Turns out when you remove the regex, you can make an ingressroute fqdn like: `*.containersteve.com` which works because Envoy accepts wildcards, but this could break delegation.